templating/embedding_controllers.rst

```diff
@@ -98,4 +98,36 @@ string syntax for controllers (i.e. **bundle**:**controller**:**action**):
             ) ?>
         </div>
 
+If your controller should be used :doc:`as a service </controller/service>`,
+you can reference it like this instead:
```

Comment:
So this is interesting :). Starting in Symfony 3.3, the `AppBundle:Article:recentArticles` syntax should also work when your controller is a service... as long as your service id matches your class name (like it does in your example). In other words, unless you've registered your service with a non-class id (e.g. `my_controller`), you should be able to just use *one* syntax, and Symfony will automatically use your controller as a service if it is registered as one (or instantiate it like normal if it is not).

Given that, what made you create this PR? Were you seeing different behavior or were you confused by something?

Cheers!